src/coreclr/vm/clrtocomcall.cpp

```diff
@@ -230,7 +230,7 @@ I4ARRAYREF SetUpWrapperInfo(MethodDesc *pMD)
 
         GCX_PREEMP();
 
-        if (pMD->IsAsyncHelperMethod())
+        if (pMD->IsAsync2Method())
```

Comment:
We use `IsAsync2Method` if we want to special case any Async2 method, most likely because the scenario cannot handle them. This would include the `Await` intrinsic, for example, if it somehow makes here.